Add FAQ

Sutrannu opened the pull request:
Merge the [FAQ repo](https://github.com/danintel/sawtooth-faq) (with history), and integrate it with the site layout.

boydjohnson commented:
@feihujiang Thanks for the contribution. I probably won't be able to merge it in until tomorrow. The sawtooth-docs build tests aren't passing based on a previous commit to one of the other repos, but there fix will be merged at least by tomorrow.

danintel committed:
Add Question/Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question/Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question/Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
RAFT Correction

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Clarify Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Clarify Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Clarify Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Raft Clarifications

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Expand Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Term

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Settings

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Settings

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Settings

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Settings

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Settings

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Settings

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add marketplace TP to prefixes

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Settings

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Videos

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Videos

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Videos

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Videos

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Videos

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Photo

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Photo

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Videos

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Mining Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Videos

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Refine Answers

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Fix Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Term

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Videos

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Refine Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Videos

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Refine Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Term

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Refine Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Refine Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Refine Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Videos

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Refine Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Videos

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Videos

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

askmish committed:
Updated zmq keygen methods for validator.toml

- Add more contextual information
- Improved indentation and spacing between lines
- Add an additional method to use the zeromq keygen tool

Signed-off-by: Ashish Kumar Mishra <ashish.k.mishra@intel.com>

danintel committed:
Format Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Format Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Format Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Refine Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

arsulegai committed:
Add more information to installation guide

Many users including me faced the issue, the error message is printed
when performing 'sudo apt-get update'. Error message states that repository
doesn't support architecture 'i386', even on 64 bit machine variant.

Signed-off-by: S m, Aruna <aruna.s.m@intel.com>

danintel committed:
Fix RST Syntax Error

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Fix Spelling Errors

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add PDO TF Prefixes

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add PDO TF Prefixes

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add PDO TF Prefixes

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Video

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Expand Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Refine Answers

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Refine Answers

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Refine Answers

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Refine Answers

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Links

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Links

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Links

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Links

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Links

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
PoET CFT

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Merge branch 'master' of https://github.com/danintel/sawtooth-faq

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Merge branch 'master' of https://github.com/danintel/sawtooth-faq

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Fix Mismatch Backquote Marks.

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Glossary

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Fix Spelling Error

Signed-off-by: danintel <daniel.anderson@intel.com>

arsulegai committed:
Info on error during apt-key addition

If the machine is behind proxy server, apt-key addition will fail.
Command for addition of apt-key needs to be changed to include
the proxy server information in it.

Signed-off-by: S m, Aruna <aruna.s.m@intel.com>

danintel committed:
Add Definition

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Fix RST Syntax Error

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Fix RST Syntax Error

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Fix RST Syntax Error

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Merge branch 'master' of https://github.com/danintel/sawtooth-faq

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Prefix

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Fix Spelling Errors

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add to Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Remove Unofficial from FAQ

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Video

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Video

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Creative Commons Attribution 4.0 International License

From
https://creativecommons.org/licenses/by/4.0/legalcode.txt
Summary:
https://creativecommons.org/licenses/by/4.0/

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Question

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Correct Validator Registry Hash Information

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Fix Validator Registry Hash Information

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Fix Validator Registry Hash Information

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Fix Validator Registry Hash Information

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Fix Validator Registry Hash Information

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Fix Validator Registry Hash Information

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Fix Validator Registry Hash Information

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Fix Validator Registry Hash Information

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add To Question

Signed-off-by: danintel <daniel.anderson@intel.com>

raneja14 committed:
Add information on permissioning

Added a question on policy creation error which is often asked.

Signed-off-by: raghav.aneja <raghav.aneja@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

danintel committed:
Add Answer

Signed-off-by: danintel <daniel.anderson@intel.com>

Sutrannu committed:
Prepare for faq repo merge

Signed-off-by: Sutrannu <bbetts@bitwise.io>

Sutrannu committed:
Merge branch 'faq' into add_faq_repo

Merging from danintel/sawtooth-faq up to commit 9d51e02

Sutrannu committed:
move and update faq files

Signed-off-by: Sutrannu <bbetts@bitwise.io>

Sutrannu committed:
Adjust formatting for site theme

Signed-off-by: Sutrannu <bbetts@bitwise.io>

danintel reviewed (approved):
It looks like it's all there. Thanks!
